error handling nginx sample

The example was copied and a null pointer is returned.

```
--- FAIL: TestNginxLatestReturn (5.82s)
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
        panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x10 pc=0x7ebcdb]

goroutine 19 [running]:
testing.tRunner.func1(0xc0000ef200)
        C:/Go/src/testing/testing.go:830 +0x399
panic(0x861de0, 0xc6f560)
        C:/Go/src/runtime/panic.go:522 +0x1c3
docker-image-bouncer.TestNginxLatestReturn(0xc0000ef200)
        C:/Users/320016328/dev/docker-image-bouncer/main_test.go:35 +0x50b
testing.tRunner(0xc0000ef200, 0x8f76b8)
        C:/Go/src/testing/testing.go:865 +0xc7
created by testing.(*T).Run
        C:/Go/src/testing/testing.go:916 +0x361
FAIL    docker-image-bouncer    6.386s
```

--- a/README.md
+++ b/README.md
@@ -21,7 +21,8 @@ import (
 	"net/http"
 	"testing"
 
-	testcontainers "github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
 )
 
 func TestNginxLatestReturn(t *testing.T) {
@@ -29,6 +30,7 @@ func TestNginxLatestReturn(t *testing.T) {
 	req := testcontainers.ContainerRequest{
 		Image:        "nginx",
 		ExposedPorts: []string{"80/tcp"},
+		WaitingFor:   wait.ForHTTP("/"),
 	}
 	nginxC, err := testcontainers.GenericContainer(ctx, testcontainers.GenericContainerRequest{
 		ContainerRequest: req,

